api4/license.go

```diff
@@ -98,6 +98,11 @@ func addLicense(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	if *c.App.Config().JobSettings.RunJobs {
+		c.App.Srv.Jobs.Workers = c.App.Srv.Jobs.InitWorkers()
+		c.App.Srv.Jobs.StartWorkers()
+	}
```

Comment:
Should we do the same when someone removes a license?

Reply:
I'll check if it's an optimization to do so.

Reply:
Ok, I just looked into this and doing the same again upon a license downgrade won't have any advantages. The `(*JobServer).InitWorkers` only does something useful here because some of the workers are `nil` so their branching statements invoke some previously un-executed code—initializing the enterprise workers.